muted-tests.yml

```diff
@@ -400,9 +397,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.ccq.MultiClusterSpecIT
   method: test {date_nanos.Bucket Date nanos by 10 minutes}
   issue: https://github.com/elastic/elasticsearch/issues/120162
-- class: org.elasticsearch.xpack.inference.InferenceCrudIT
-  method: testGetServicesWithCompletionTaskType
-  issue: https://github.com/elastic/elasticsearch/issues/119959
```

Comment:
This test doesn't live in `InferenceCrudIT` anymore. It lives here: https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/inference/qa/inference-service-tests/src/javaRestTest/java/org/elasticsearch/xpack/inference/InferenceGetServicesIT.java#L112

I believe the issue was related to the EIS feature flag which no longer exists.